cli/src/main/resources/application.yml

```diff
@@ -31,7 +31,7 @@ address:
     queryTimeout: 2000
 
 connection:
-  timeout: 1000
+  retryInterval: 1000
```

Comment:
looks like you can pull this up to the top level

Reply:
It makes sense and I pulled it up.